### ISSUE TITLE
Create asset and then get events returns empty results

### ISSUE THREAD

**Issue by asebak** [ambrosus/sdk-javascript#3]
If you create an asset with this payload:

    [
      {
        "type": "ambrosus.asset.info",
        "name": "PURE DARK CHOCOLATE BAR 92%",
        "images": {
          "default": {
            "url": "http://imageurlgoeshere.com/file.extension"
          }
        },
        "size": "2.64 oz.",
        "Product Information": {
          "attributes": "No-GMOs, Vegan, Gluten Free, Kosher, Soy Free",
          "ingredients": "Organic cocoa beans, organic sugar, organic cocoa butter",
          "Brand": "Madecasse"
        },
        "Batch Information": {
          "Origin": "Madagascar"
        }
      }
    ];

and then you do getEvents with only the asset id as the only options in the body, it returns no results.  I would expect to the above payload since it's event type is ambrosus.asset.info

So to summarize the behavior: does creating an asset automatically create an event? I was expecting it to happen, so maybe it's by design?

**Comment by lazar-eric** [comment]
Hi, please take a look at: https://github.com/ambrosus/sdk-javascript/blob/master/examples/demo-assets/chocolate.json. Each object in the array passed to createAsset() has to have content property, and inside of it data property, and inside of data goes the array of event objects, one of them being let's say 'ambrosus.asset.info' (from your example).

So this would be correct:

`
[
	{
		"content": {
			"data": [
				{
					"type": "ambrosus.asset.info",
					"name": "PURE DARK CHOCOLATE BAR 92%",
					"images": {
						"default": {
							"url": "http://imageurlgoeshere.com/file.extension"
						}
					},
					"size": "2.64 oz.",
					"Product Information": {
						"attributes": "No-GMOs, Vegan, Gluten Free, Kosher, Soy Free",
						"ingredients": "Organic cocoa beans, organic sugar, organic cocoa butter",
						"Brand": "Madecasse"
					},
					"Batch Information": {
						"Origin": "Madagascar"
					}
				}
			]
		}
	}
]
`
